lib/client/db/dbcmd/dbcmd.go

```diff
@@ -505,6 +510,17 @@ func (c *CLICommandBuilder) getSnowflakeCommand() *exec.Cmd {
 	return cmd
 }
 
+func (c *CLICommandBuilder) getCassandraCommand() (*exec.Cmd, error) {
+	args := []string{
+		"-u", c.db.Username,
+		c.host, strconv.Itoa(c.port),
```

Comment:
do we support --db-name (e.g. `-k KEYSPACE`)?  Also the default timeouts seems pretty small, not sure if we want to bump up a bit)

Reply:
> Also the default timeouts seems pretty small, not sure if we want to bump up a bit)
The default times are: 
```
  --connect-timeout=CONNECT_TIMEOUT
                        Specify the connection timeout in seconds (default: 5
                        seconds).
  --request-timeout=REQUEST_TIMEOUT
                        Specify the default request timeout in seconds
                        (default: 10 seconds).
 ```
I think that 5s and 10s should be enough. 

> do we support --db-name (e.g. -k KEYSPACE)? 

I think that cqlsh interaction works similar to mysql where we don't support --db-name because a db can be freely change in query like: ` SELECT * FROM otherkeyspace.tablename` and there is no easy whey to support db-name without parsing the sql query.